views/personal/maritalStatus-edit.pug

```diff
@@ -0,0 +1,45 @@
+extends ../base
+
+block variables
+  -var title = 'Change your Marital Status'
+
+block content
+
+  h1 #{title}
+
+  div
+    p Please choose the correct status from the options listed below.
+
+    checked=(data && data.personal.maritalStatus.toLowerCase()=='married')
```

Comment:
I think the `-var ` prefix is necessary here if you're using it — which I'm not sure that you are. 🤔 

Currently, this creates an empty `<checked></checked>` tag under the paragraph.

Reply:
that should be there and not sure how it go there 🤔 
good catch! will remove